lib/modules/manager/terraform/extract.spec.ts

```diff
@@ -401,8 +401,8 @@ describe('modules/manager/terraform/extract', () => {
 
     it('extracts docker resources', async () => {
       const res = await extractPackageFile(docker, 'docker.tf', {});
-      expect(res?.deps).toHaveLength(8);
-      expect(res?.deps.filter((dep) => dep.skipReason)).toHaveLength(5);
+      expect(res?.deps).toHaveLength(6);
```

Comment:
These two [fixtures](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/terraform/__fixtures__/docker.tf#L46-L54) are filtered out directly by the path algorithm. 

One removed dep is `not_supported_resource` which if now filtered out by not existing in [generic_image_resource](https://github.com/renovatebot/renovate/pull/19269/files#diff-ec2c22cb2f2a06cdd97914498107e69cdb371c58e5f2025f2b7114f8256a9222R38). 

The second missing dep with `skipReason` is the `docker_service` which is removed because the path is not followed till the end as the sub blocks are not defined.